CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and adheres to [Semantic Versioning](http://semver.org) as of version 1.0.0.
```

Comment:
We also don't really define a public API right now, which is a requirement for SemVer.